opentasks/src/main/java/org/dmfs/tasks/utils/AbstractStringCharSequence.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2017 dmfs GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dmfs.tasks.utils;
+
+/**
+ * Base class for {@link CharSequence}s that use a {@link String} delegate value.
+ *
+ * @author Gabor Keszthelyi
+ */
+public abstract class AbstractStringCharSequence implements CharSequence
```

Comment:
This class is a bit weird. It's essentially an adapter for String to CharSequence, which doesn't make much sense, because Strings already are CharSequences.
I'll accept this for now, but please create a story to replace this by something like `Lazy` which seems to be the intent behind this.